en-US/Players_Guide/03_Creating_a_Character/Level_Advancement/Multiclassing.md

```diff
@@ -0,0 +1,139 @@
+Use this total to determine your spell slots by consulting the Multiclass Spellcaster table.
+
+If you have more than one spellcasting class, this table might give you spell slots of a level that is higher than the spells you know or can prepare.
+You can use those slots, but only to cast your lower-level spells.
+If a lower-level spell that you cast, like _Burning Hands_, has an enhanced effect when cast using a higher-level slot, you can use the enhanced effect, even though you don’t have any spells of that higher level.
+
+For example, if you are the aforementioned Ranger 4/Wizard 3, you count as a 5th-level character when determining your spell slots: you have four 1st-level slots, three 2nd-level slots, and two 3rd-level slots.
+However, you don’t know any 3rd-level spells, nor do you know any 2nd-level Ranger spells.
+You can use the spell slots of those levels to cast the spells you do know—and potentially enhance their effects.
+
+**Pact Magic.**
+
+If you have both the Spellcasting class feature and the Pact Magic class feature from the Warlock class, you can use the spell slots you gain from the Pact Magic feature to cast spells you know or have prepared from classes with the Spellcasting class feature, and you can use the spell slots you gain from the Spellcasting class feature to cast Warlock spells you know.
+
+<!-- TODO Add the multiclass spellcaster spell slots per spell level table -->
+
+###### Alignment
+
+A creature’s alignment in the game world describes its moral and personal attitudes.
+It’s a combination of morality (good, evil, or neutral) and attitudes toward society and order (lawful, chaotic, or neutral).
+Thus, nine alignments exist.
+
+These brief descriptions outline the typical behavior of a creature with an alignment.
+Individuals may vary, and few are consistently faithful to their alignment’s precepts.
+
+**Lawful good (LG)** creatures can be counted on to do the right thing as expected by society.
+\
+**Neutral good (NG)** folk do the best they can to help others according to their needs.
+\
+**Chaotic good (CG)** creatures act as their conscience directs, with little regard for what others expect.
+\
+**Lawful neutral (LN)** individuals act in accordance with law, tradition, or personal codes.
+\
+**Neutral (N)** is the alignment of those who prefer to steer clear of moral questions and don’t take sides, doing what seems best at the time.
+\
+**Chaotic neutral (CN)** creatures follow their whims, holding their personal freedom above all else.
+\
+**Lawful evil (LE)** creatures methodically take what they want, within the limits of a code of tradition, loyalty, or order.
+\
+**Neutral evil (NE)** is the alignment of those who do whatever they can get away with, without compassion or qualms.
+\
+**Chaotic evil (CE)** creatures act with arbitrary violence, spurred by their greed, hatred, or bloodlust.
+
+<!-- TODO add the languages section -->
```

Comment:
There are some open comments here; I can either wait for those to be resolved in the Google Doc, or transfer them to GitHub as well. Do you have a preference?